src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -383,6 +384,9 @@ protected JSObject getJsObject() {
     return (JSObject) engine.executeScript("window");
   }
 
+  protected void onWebViewLoaded() {
+  }
```

Comment:
Leere Methoden in abstrakten Klassen sind aus meiner Sicht selbsterklärend. Du hast doch selbst eingefordert, keine unnötigen Kommentare zu schreiben.

Deinem Kommentarvorschlag kann ich nicht 100% zustimmen. "to be overriden" impliziert, dass man das tun soll. Das steht erbenden Klassen aber völlig frei. Wenn ich das erzwingen wollte, hätte ich die Methode abstract deklariert.

Wäre also eher:
```
protected void onWebViewLoaded() {
  // Default implementation does nothing, can be overridden by subclass.
}
```